### PR TITLE
Fix coreos login IP and instructions in README

### PR DESCRIPTION
**Pull request by benbp**

### CODE REVIEW

**Comment by benbp** [comment]
@heckj

**Comment by benbp** [comment]
@DavidjohnBlodgett

File: example/README.md [hunk 1]
```diff
@@ -140,11 +140,9 @@ installed. The installation workflow included in the RackHD system installs
 with a default SSH key that's included in our repositories. From the `example`
 directory, you should be able to log in using:
 
-`vagrant ssh`
+`vagrant ssh`:
 
-    cp ~/src/on-http/data/rackhd_rsa ~/.ssh/id_rsa
-    chmod 400 ~/.ssh/id_rsa
-    ssh -i ~/src/on-http/data/rackhd_rsa core@172.32.128.2
+    ssh -i ~/src/on-http/data/rackhd_rsa core@172.31.128.2
```
Comment (heckj) [inline example/README.md]: Unfortunately, I don't think this'll work - SSH won't let you use the key without more constrained permissions, and git checks it out as "644" default. So at the minimu, you need to `chmod 400 ~/src/on-http/data/rackhd_rsa`

**Comment by tldavies** [comment]
+1

File: example/README.md (hunk 1 above)
Comment (benbp) [inline example/README.md]: So, should we just do the first two steps and then `ssh core@...` instead of `ssh -i` ?

**Comment by heckj** [comment]
Moving into a local .ssh/ may make other demo steps easier later. Could even do as part of an ansible vagrant setup role too

**Comment by benbp** [comment]
Sounds good, will make the change.

**Comment by heckj** [comment]
+1 (thanks @benbp )

**Pull request merged**